### PR TITLE
Small improvments to profile dialog

### PR DESCRIPTION
**Pull request by Midhil457**
- Make tabs always on top
- Add empty message if no  contributions are made
- Disable Assets Accordian

profile-comparision

profile-2

### CODE REVIEW

File: newIDE/app/src/Profile/ContributionsDetails.js [hunk 1]
```diff
@@ -69,18 +70,24 @@ export const ExamplesAccordion = ({
...
-                example.previewImageUrls ? example.previewImageUrls[0] : ''
-              }
-            />
-          ))}
-        </Column>
+        {examples.length === 0 && (
+          <EmptyMessage>You haven't contributed any examples</EmptyMessage>
+        )}
+
+        {examples.length !== 0 && (
```
Comment (4ian) [inline newIDE/app/src/Profile/ContributionsDetails.js:77]: Use `<Trans>` to mark strings to be translated in JSX, and add a period at the end of the sentence.
Also nitpicking: prefer using the ternary to make it explicit this a choice between one or the other :) 

```suggestion
        {examples.length === 0 ? (
          <EmptyMessage><Trans>You haven't contributed any examples.</Trans></EmptyMessage>
        ) : (
```

File: newIDE/app/src/Profile/ContributionsDetails.js [hunk 2]
```diff
@@ -114,16 +121,22 @@ export const ExtensionsAccordion = ({
...
-          {extensions.map(extension => (
-            <ContributionLine
-              key={extension.name}
-              shortDescription={extension.shortDescription}
-              fullName={extension.fullName}
-              previewIconUrl={extension.previewIconUrl}
-            />
-          ))}
-        </Column>
+        {extensions.length === 0 && (
```
Comment (4ian) [inline newIDE/app/src/Profile/ContributionsDetails.js:124]: Same here ;)

**Comment by 4ian** [comment]
Nice! Minor comments and then this is good to go :)

**Comment by Midhil457** [comment]
Made the changes :)

**Review by 4ian — APPROVED** [review]
Nice 👍

**Pull request merged**